decomp/semantics/uds.py

```diff
@@ -54,18 +56,24 @@ class UDSCorpus(PredPattCorpus):
 
     def __init__(self,
                  graphs: Optional[PredPattCorpus] = None,
+                 docs: Dict[str, 'UDSDocument'] = None,
                  annotations: List['UDSAnnotation'] = [],
                  version: str = '1.0',
```

Comment:
I think we want to bump the default version to `'2.0'`. We should retain the `1.0` directory within `data/` (though based on comments below it may need to be restructured) and add a `2.0` directory. One thing I've been wondering is if we want to actually have two subdirectories of each version: `normalized` and `raw`, rather than viewing `raw` as outside the versioning system as we are currently doing. Then, `raw` would just have whichever raw annotations went into deriving the normalized annotations for that version (possibly with additional annotations in the case where we don't want to commit to a single normalized value, as for temporal relations). This additional structure would allow us to release additional processed versions of the normalized annotations—e.g. the optimal clustering labels we have been discussing for the UDS-EventStructure paper.

Reply:
I think this makes sense. Would you suggest simply copying over the `1.0` annotations into the subdirectory for `2.0` then (for now, at least)?

Reply:
Yeah. When we update the normalization (using our clustering model rather than the previous models), we can update the 2.0 annotations.